dask/dataframe/core.py

```diff
@@ -1079,6 +1080,53 @@ def sample(self, n=None, frac=None, replace=False, random_state=None):
         return new_dd_object(merge(self.dask, dsk), name,
                              self._meta, self.divisions)
 
+    def to_dask_array(self, lengths=None):
+        """Convert a dask DataFrame to a dask array.
+
+        Parameters
+        ----------
+        lengths : bool or Sequence of ints, optional
```

Comment:
Is this keyword name OK? Or should it be just be chunks? It's not quite the same, since we also have the optional "compute for me" behavior.